Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/CustomTabUtil.java

```diff
@@ -0,0 +1,17 @@
+package com.mentalmachines.droidcon_boston.utils;
+
+import android.content.Context;
+import android.net.Uri;
+import android.support.customtabs.CustomTabsIntent;
+import com.mentalmachines.droidcon_boston.R;
+
+public class CustomTabUtil {
+
+  public void loadUriInCustomTab(Context context, String uriString) {
```

Comment:
should be static

Reply:
Will fix this in the new branch and PR. This method is unused in this PR and exists solely to preserve what the other dev had written in the first place.

Reply:
Fixed in recent commit pushed to this PR